### ISSUE TITLE
Toy Datasets for Gaussianization Flows method

### ISSUE THREAD

**Issue by jejjohnson** [IPL-UV/rbig_jax#16]
Need to do some toy experiments on the classic datasets with the Scikit-learn cluster data.

**Datasets**

* [x] Classic RBIG
* [x] [Circles](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_circles.html#sklearn.datasets.make_circles)
* [x]  [Moons](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_moons.html#sklearn.datasets.make_moons)
* [x] [S-Curve](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_s_curve.html#sklearn.datasets.make_s_curve)
* [x] [Blobs](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_blobs.html#sklearn.datasets.make_blobs)
* [x] [Swiss Roll](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_swiss_roll.html#sklearn.datasets.make_swiss_roll)

**ToDo**

* [x] Scripts
* [x] Loss Metrics (nll, info reduction)
* [x] Plots (latent, original)
* [x] Sampling
* [x] Density Estimation
* [x] Wandb integration ([example](https://docs.wandb.ai/integrations/jax))
